http/handlers.go

```diff
@@ -98,7 +98,7 @@ func DecodeRequestWithHeaders(_ context.Context, req *http.Request, pathParams m
 	request["host"] = req.Host
 
 	buf := new(bytes.Buffer)
-	buf.ReadFrom(req.Body)
+	_, _ = buf.ReadFrom(req.Body)
```

Comment:
?

Reply:
Reverted to keep sonar code smell and process it correctly when we'll have time